libs/watt/src/lib/components/datetime-field/watt-datetime-field.component.ts

```diff
@@ -0,0 +1,204 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  ChangeDetectionStrategy,
+  Component,
+  computed,
+  forwardRef,
+  inject,
+  input,
+  output,
+  signal,
+  viewChild,
+  ViewEncapsulation,
+} from '@angular/core';
+import {
+  ControlValueAccessor,
+  FormControl,
+  NG_VALUE_ACCESSOR,
+  ReactiveFormsModule,
+} from '@angular/forms';
+import { MatCalendar } from '@angular/material/datepicker';
+import { MaskitoDirective } from '@maskito/angular';
+import { maskitoDateTimeOptionsGenerator } from '@maskito/kit';
+import { map } from 'rxjs';
+import { dayjs } from '@energinet-datahub/watt/date';
+import { WattFieldComponent } from '../field';
+import { WattButtonComponent } from '../button/watt-button.component';
+import { outputFromObservable } from '@angular/core/rxjs-interop';
+import { WattLocaleService } from '@energinet-datahub/watt/locale';
+
+const DA_FILLER = 'dd-mm-åååå, tt:mm';
+const EN_FILLER = 'dd-mm-yyyy, hh:mm';
```

Comment:
Good point on being consistent, however I do not understand why the time part should be capitalized and the date not? That is just inconsistency within the filler itself 😄 I took inspiration from the maskito examples, which use all lowercase. Maybe we should just update the timepicker instead (which is about to become "obsolete" for the time being, since this component supersedes the last component that was using it).

Reply:
I think the "HH:MM" format comes partly from the formatting library, where `HH` means `00 01 ... 22 23` values and partly from "Why use capital HH with lower-case mm, let's have both capital"-reasoning. Because `MM` means something else when formatting dates. So I think the lower-case is more correct.

Reply:
Yeah, as developers we are used to the format strings, but no users are ever going to understand the difference between `dd-MM-yyyy, HH:mm` and `dd-mm-yyyy, hh:mm` 😄 It will just look off to them I think. We could also consider using all capitals instead (`DD-MM-YYYY, HH:MM`)? 🙂

Reply:
I think lowercase is fine until someone comes with better arguments for doing something else. We can adjust other components as follow-up PRs.